diff -up openjdk/src/java.base/share/conf/security/java.security.1~ openjdk/src/java.base/share/conf/security/java.security
--- openjdk/src/java.base/share/conf/security/java.security.1~	2020-10-19 10:54:38.224370673 +0200
+++ openjdk/src/java.base/share/conf/security/java.security	2020-10-19 10:56:08.598815470 +0200
@@ -78,6 +78,7 @@ security.provider.tbd=SunMSCAPI
 security.provider.tbd=Apple
 #endif
 security.provider.tbd=SunPKCS11
+#security.provider.tbd=SunPKCS11 ${java.home}/lib/security/nss.cfg
 
 #
 # A list of preferred providers for specific algorithms. These providers will
